### ISSUE TITLE
Omitting babel-plugin-transform-es2015-classes

### ISSUE THREAD

**Issue by shtse8** [Vincit/objection.js#572]
https://github.com/Vincit/objection.js/tree/887bc424145dd7ca1639063b8224dedf935f5960/examples/express-es7
As the document stating that 
> For this reason we need to omit the babel-plugin-transform-es2015-classes plugin. There is no way to omit plugins from presets.

We can exclude the plugin in babel as follow in babel-preset-env:
```
["env", {
      "targets": {
        "browsers": ["> 1%", "last 2 versions", "not ie <= 8"]
      },
      exclude: [
        "babel-plugin-transform-es2015-classes"
      ]
    }],
```

I think we can have a note in readme for others.

**Comment by koskimas** [comment]
You are right, I should update the example project to use the "env" plugin. By the way, this should be enough:

```
{
  "presets": [
    ["env", {
      "targets": {
        "node": "current"
      }
      }],
    "stage-1"
  ],
  "plugins": []
}
```

**Comment by AnianZ** [comment]
I tried to update the examples `package.json` to the suggested presets as a first contribution to the project. I hope it is working as expected.

**Comment by koskimas** [comment]
@AnianZ Fixed this in 3867ab5. Thank you!
